stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (C) 2011 Julius Network Solutions SARL <contact@julius.fr>
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class StockMoveLocationWizardLine(models.TransientModel):
+    _name = "wiz.stock.move.location.line"
+
+    move_location_wizard_id = fields.Many2one(
+        string="Move location Wizard",
+        comodel_name="wiz.stock.move.location",
+        ondelete="cascade",
+        required=True,
+    )
+    product_id = fields.Many2one(
+        string="Product",
+        comodel_name="product.product",
+        required=True,
+    )
+    origin_location_id = fields.Many2one(
+        string='Origin Location',
+        comodel_name='stock.location',
+    )
+    destination_location_id = fields.Many2one(
+        string='Destination Location',
+        comodel_name='stock.location',
+    )
+    product_uom_id = fields.Many2one(
+        string='Product Unit of Measure',
+        comodel_name='product.uom',
+    )
+    lot_id = fields.Many2one(
+        string='Lot/Serial Number',
+        comodel_name='stock.production.lot',
+        domain="[('product_id','=',product_id)]"
+    )
+    move_quantity = fields.Float(
+        string="Quantity to move",
+        digits=dp.get_precision('Product Unit of Measure'),
+    )
+    max_quantity = fields.Float(
+        string="Maximum available quantity",
+        digits=dp.get_precision('Product Unit of Measure'),
+    )
+    custom = fields.Boolean(
+        string="Custom line",
+        default=True,
+    )
+
+    @api.model
```

Comment:
this is not api.model as you are using `self.product_uom_id`
I'd make it a static method and pass rounding factor:
```py
@staticmethod
def _compare(qty1, qty2, precision):
```
Alternative is to use @api.multi + ensure_one or... 
do not use any decorator at all as it's unlikely that you gonna call this via rpc.
In any case, is going to be broken in a very clear way if you call it on non recordset or multiple records.

Reply:
PR updated (I used the @staticmethod, I'm not sure to understand the link with/without decorator and RPC interface, I though that by default @api.multi was applied by Odoo).